post_installation_configuration/preparing-for-users.adoc

```diff
@@ -118,18 +118,16 @@ include::modules/authentication-kubeadmin.adoc[leveloffset=+1]
 include::modules/authentication-remove-kubeadmin.adoc[leveloffset=+2]
 
 [id="post-install-image-configuration-resources"]
-== Image configuration resources
+== Image configuration
 Understand and configure image registry settings.
 
 include::modules/images-configuration-parameters.adoc[leveloffset=+2]
 
 include::modules/images-configuration-file.adoc[leveloffset=+2]
 
-include::modules/images-configuration-cas.adoc[leveloffset=+3]
-
-include::modules/images-configuration-insecure.adoc[leveloffset=+3]
+For more information on the allowed, blocked, and insecure registry parameters, see xref:../openshift_images/image-configuration.adoc#images-configuration-file_image-configuration[Configuring image registry settings].
```

Comment:
Because this assembly heavily favors includes over xrefs, I suggest _considering_ using an include instead. (I assume that you're happy with the modules that you're removing.)

Reply:
@kalexand-rh I would need to add 2 includes, for a total of 4, to get all of the topics in the linked topic. That seemed  a bit excessive, as the topic is already quite long. That one section should give enough info for folks to get started. The insecure and short name are optional. Perhaps @ahardin-rh can weigh in when she gets back?